cmake/gridformat-config.cmake.in

```diff
@@ -23,7 +23,18 @@ if (@MPI_FOUND@)
     find_dependency(MPI)
 endif ()
 if (@GRIDFORMAT_HAVE_HIGH_FIVE@)
-    find_dependency(HighFive)
+    set(HDF5_PREFER_PARALLEL @HIGHFIVE_PARALLEL_HDF5@)
+    if (GRIDFORMAT_NO_HIGHFIVE_HINTS)
+        find_dependency(HighFive)
+    else ()
+        find_dependency(
+            HighFive
+            HINTS
+                ${CMAKE_CURRENT_LIST_DIR}/../deps/HighFive  # for linking to the build tree
+                @PACKAGE_GRIDFORMAT_INSTALL_CMAKE_DATA_DIR@/../HighFive  # in case HighFive was installed alongside gridformat
+                @HighFive_ROOT@  # in case HighFive Root directory was given
```

Comment:
isn't `module_ROOT` automatically checked before HINTS? I think that always takes priority : https://cmake.org/cmake/help/latest/command/find_package.html#command:find_package

Reply:
but so this gets expanded when gridformat is configured... So you may configure gridformat like this:

```bash
cmake -DHighFiveRoot=/somehwere -DCMAKE_INSTALL_PREFIX=/somewhereelse ...
```

and then link downstream against gridformat like

```bash
cmake -Dgridformat_ROOT=/somewhereelse
```

and then it doesn't find `HighFive` in `/somewhere`. With this hint, the root dir upon gridformat configuration is hardcoded in the installed config file.

Reply:
I ran into this case in a minimal working example with preinstalled highfive (in a non-standard location)